boards/arm/lpcxpresso54114/extra.cmake

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) 2019, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# This cmake extra file is used for definitions that are custom to this board
+
+
+# Suppress "simple_bus_reg" on LPC boards as all GPIO ports use the same register.
+set(EXTRA_DTC_FLAGS "-Wno-simple_bus_reg")
```

Comment:
11 lines for 1 line of code might be a bit long. Can make it

```cmake
# Copyright (c) 2019, NXP
# SPDX-License-Identifier: Apache-2.0

# Suppress "simple_bus_reg" on LPC boards as all GPIO ports use the same register
set(EXTRA_DTC_FLAGS "-Wno-simple_bus_reg")
```

if the copyright/license stuff needs to be there.

Reply:
agree